bin/dune

```diff
@@ -1,4 +1,4 @@
 (executable
  (public_name gtirb_semantics)
  (name main)
- (libraries gtirb_semantics asli.libASL))
+ (libraries base64 yojson gtirb_semantics asli.libASL))
```

Comment:
asli.libASL -> asli

Reply:
It fails to build saying can't find the library with just `asli`, but it compiles without anything listed as well.

Reply:
It looks like this should stay as asli.libASL. It does work without (much to my surprise), but let's be specific.